_rules/role-attribute-valid-value-674b10.md

```diff
@@ -170,10 +170,15 @@ This `role` attribute is only [ASCII whitespace][].
 
 #### Inapplicable Example 5
 
-This `role` attribute is specified on an element which is [programmatically hidden][].
+These `role` attributes are specified on elements which are [programmatically hidden][].
 
 ```html
 <div aria-hidden="true" role="banner">Some Content</div>
+<div hidden role="banner">Some Content</div>
+<div hidden="until-found" role="banner">Some Content</div>
+<div style="display: none" role="banner">Some Content</div>
+<div style="visibility: hidden" role="banner">Some Content</div>
+<div style="content-visibility: hidden" role="banner">Some Content</div>
```

Comment:
I'm not sure these are all necessary. We're not writing unit tests for the `programmatically hidden` definition here. For what is necessary we need separate examples. We should only have one test per test case.